src/status_im2/subs/wallet/wallet_test.cljs

```diff
@@ -1,8 +1,9 @@
 (ns status-im2.subs.wallet.wallet-test
   (:require [cljs.test :refer [is testing use-fixtures]]
             [re-frame.db :as rf-db]
-            status-im2.subs.root
+            [status-im2.subs.root]
```

Comment:
What's the difference? 🤔

Reply:
No difference, I can revert it, just wanted to  have the same style in all the requires (using `[]`)

Reply:
no need, I was just wondering